set.mm

```diff
@@ -153948,6 +154055,29 @@ are used instead of sets because their representation is shorter (and more
       DXRYEUVEUVFXPXSXQBCDAUVJUVGYEUVK $.
   $}
 
+  $( The number of elements of a finite function restricted to a subset of its
+     domain.  (Contributed by AV, 15-Dec-2021.) $)
+  hashres $p |- ( ( Fun A /\ A e. Fin /\ B C_ dom A )
+                  -> ( # ` ( A |` B ) ) = ( # ` B ) ) $=
+    ( wfun cfn wcel cdm wss w3a cres chash cfv wceq funres 3ad2ant1 wb 3ad2ant2
+    finresfin hashfun syl mpbid ssdmres biimpi 3ad2ant3 fveq2d eqtrd ) ACZADEZB
+    AFGZHZABIZJKZUJFZJKZBJKUIUJCZUKUMLZUFUGUNUHBAMNUIUJDEZUNUOOUGUFUPUHBAQPUJRS
+    TUIULBJUHUFULBLZUGUHUQBAUAUBUCUDUE $.
+
+  $( The number of elements of a finite function expressed by a restriction.
+     (Contributed by AV, 15-Dec-2021.) $)
+  hashreshashfun $p |- ( ( Fun A /\ A e. Fin /\ B C_ dom A )
```

Comment:
The number of elements of a finite function is equal to the sum of the number of elements with argument (first component of the ordered pair) in some subset of its domain and of the number of elements with argument not in that subset.

Reply:
I think the proposed one-to-one translation of the formula is correct, but hard to understand. I remember that in several similar cases provided by me Norm asked me to revise such comments and not to try to exactly describe the formula. Therefore, I would not chnage the comment. Are there other opinions?

Reply:
I agree with Alexander (and Norm), especially here where the statement of the theorem is particularly short and easy to understand by itself already.